Database/Corrections/cataQuestFixes.lua

```diff
@@ -4454,9 +4489,17 @@ function CataQuestFixes.Load()
             [questKeys.preQuestSingle] = {26436},
             [questKeys.objectives] = {{{43115,nil,Questie.ICON_TYPE_INTERACT}}},
         },
+        [26440] = { -- Clingy
+            [questKeys.triggerEnd] = {"Pebble brought to crystal formation",{[zoneIDs.DEEPHOLM]={{29.45,47.47}}}},
+            [questKeys.objectives] = {},
+            [questKeys.preQuestSingle] = {26439},
+        },
         [26442] = { -- Rock Lobster
             [questKeys.exclusiveTo] = {29325,29321,29323,29324,29342,29343,29344,29347,29350,29359,26414,26420,26488,26536},
         },
+        [26447] = { -- Diabolical Plans
+            [questKeys.startedBy] = {nil,nil,{23798}},
+        },
```

Comment:
What did you want to do with this one here? :)

Reply:
Add a marker similar to that for the Ashenvale hunt quests (e.g. Shadumbra)